### PR TITLE
Model on frontend

### PR DESCRIPTION
**Pull request by Caspersn2**

### CODE REVIEW

File: WebServer/frontend/src/app/model-changer/model-changer.component.html [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
+        <label for="types">Chose which model to upload</label>
+        <select class="form-control" id="types" (change)="choseType($event)">
+            <option>Face</option>
+            <option>License plate</option>
+        </select>
+    </div>
+
+    <br>
+
+    <input type="file" id="file" (change)="handleFileInput($event.target.files)" accept=".h2,.pb">
```
Comment (AnneEjsing) [inline WebServer/frontend/src/app/model-changer/model-changer.component.html:13]:
```suggestion
    <input type="file" id="file" (change)="handleFileInput($event.target.files)" accept=".h5,.pb">
```

File: WebServer/microservices/db/camera.py [hunk 2]
```diff
@@ -126,3 +126,43 @@ async def camera_userlist(request):
...
+    query = """
+    UPDATE cameras 
+    SET model_licens = %s
+    WHERE camera_id = %s
+    RETURNING *;
+    """
+
+    result, error = dbresolver.execute_query(query, model_licens, id)
+    if error: return web.Response(text=error,status=500)
+    return dbresolver.has_one_result(result, "There is no camera with that id.", 404)
```
Comment (lrecht) [inline WebServer/microservices/db/camera.py:168]: Should probably be tested

File: WebServer/microservices/dispatcher/service.py [hunk 3]
```diff
@@ -248,7 +248,7 @@ async def upload_model(request):
         # Data cannot just be forwarded. File need to be sent using the format below.
         model = data['file']
         files = {'file': (model.filename, model.file, model.content_type, model.headers)}
-        data={'camera_id':data['camera_id']}
+        data={'camera_id':data['camera_id'], 'type':data['type']}
         response = requests.post(endpoint,data=data, files=files)
```
Comment (lrecht) [inline WebServer/microservices/dispatcher/service.py:252]: maybe need to update api tests

File: WebServer/microservices/model-changer/swap-models.py [hunk 4]
```diff
@@ -24,7 +25,16 @@ async def remote_change_ml(request):
...
+    response = requests.post(url, files=files, data={ 'type': model_type})
+
+    if (response.status_code == 200):
+        # The model has been updated on the nano. Now update in database
+        if (model_type == 'face'):
+            model_face = model.filename
+            requests.put(dbresolver + "camera/update_models/face", headers={'Content-type': 'application/json'}, json={'id': camera_id,'model_face': model_face})
+        elif (model_type == 'license_plate'):
+            model_licens = model.filename
+            requests.put(dbresolver + "camera/update_models/licens", headers={'Content-type': 'application/json'}, json={'id': camera_id,'model_licens': model_licens})
```
Comment (lrecht) [inline WebServer/microservices/model-changer/swap-models.py:37]: Update api tests

**Pull request merged**